R/utils_render_common.R

```diff
@@ -478,7 +478,7 @@ create_summary_rows <- function(n_rows,
 
       summary_df <-
         list_of_summaries$summary_df_display_list[[group]] %>%
-        dplyr::select(-groups) %>%
+        dplyr::select(-group) %>%
```

Comment:
Use `-.env$group`

Reply:
I was mistaken during our code review, this should instead be `-.data$group`.